icu4j/main/tests/core/src/com/ibm/icu/dev/test/number/NumberFormatterApiTest.java

```diff
@@ -55,6 +39,21 @@
 import com.ibm.icu.util.MeasureUnit;
 import com.ibm.icu.util.NoUnit;
 import com.ibm.icu.util.ULocale;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.text.FieldPosition;
+import java.text.Format;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
```

Comment:
https://sites.google.com/site/icuprojectuserguide/dev/codingguidelines
"The sort order of import statements is java / javax / org / com."

Reconfigure intellij, or let eclipse handle it?
(For clang-format, I have this: `JavaImportGroups: ['java', 'javax', 'org', 'com']` )

Reply:
Please fix the import order.

Reply:
Thanks!